components/layouts/main/footer.tsx

```diff
@@ -1,3 +1,16 @@
-export default function Footer() {
-  return <footer className="border-t-2 border-t-pnc-grey">Footer</footer>;
+type Props = {
+  updateDate: string;
+  versionNumber: string;
+};
+
+export default function Footer({updateDate, versionNumber}: Props) {
+
+  return (
+    <div className="absolute bottom-6 right-0 left-0 flex justify-center h-[4.25rem] border-t-2 border-pnc-grey mx-[12.5rem] mt-16">
```

Comment:
Could you keep the `footer` element instead of `div`, please? The rendered html would be slightly more semantic.

Fixing the footer in an absolute position seems a bit too early. We do not know it yet wether the footer should always be visible. I would recommend leaving it as flow content for now.